documentation/content/en/books/handbook/zfs/_index.adoc

```diff
@@ -1335,11 +1335,13 @@ For systems using EFI to boot, execute the following command:
 
 [source,shell]
 ....
-# gpart bootcode -p /boot/boot1.efifat -i 1 ada1
+# mount_msdosfs /dev/ada1p1 /boot/efi
```

Comment:
By default this will automatically be mounted (bsdinstall puts it in fstab) and so will fail with `mount_msdosfs: /dev/ada1p1: Device busy`. If you need to mount it manually for whatever reason, mount -t msdosfs is better than mount_msdosfs (which is a bit of an implementation detail).

Reply:
Correction, it "may fail." You're assuming the disks were created with bsdinstall. There are more than a few FreeBSD disks roaming about that were installed with sysinstall or were manually partitioned to match disks installed long ago.

Reply:
I realise. I deliberately said *by default* at the start of the sentence to fully acknowledge there are other ways to get a working FreeBSD system. That is, by default it will fail. That doesn't mean it will always fail.

Reply:
How about this?

```sh
mount -t msdosfs -u /dev/ada1p1 /boot/efi
```